replay-app/package.json

```diff
@@ -37,18 +37,18 @@
   "scripts": {
     "build:frontend": "nuxt-ts build",
     "build:backend-paths": "tscpaths -p tsconfig.backend.json -s ./ -o ../build/replay-app",
-    "build:backend": "tsc -b tsconfig.backend.json && yarn build:backend-paths && cp package.json ../build/replay-app",
+    "build:backend": "tsc -b tsconfig.backend.json && yarn build:backend-paths && shx cp package.json ../build/replay-app",
     "build:icons": "electron-icon-builder -f --input=./frontend/static/AppIcon.appiconset/icon_512@2x.png --output=../build/replay-app/build",
-    "build:dist": "yarn build:frontend && yarn build:backend && yarn build:icons && yarn logs:reset",
+    "build:dist": "yarn build:frontend && yarn build:backend && yarn build:icons",
     "build:pack": "yarn build:dist && cd ../build/replay-app && yarn && electron-builder -mwl",
     "build": "yarn build:dist",
     "gh:publish": "yarn build:dist && cp .env ../build/replay-app && cd ../build/replay-app && yarn && export $(cat .env | xargs) && electron-builder -mwl -p=onTagOrDraft",
-    "clean": "rm -rf ../build/replay-app/static && tsc -b --clean tsconfig.backend.json && tsc -b --clean tsconfig.json",
+    "clean": "shx rm -rf ../build/replay-app/static && tsc -b --clean tsconfig.backend.json && tsc -b --clean tsconfig.json",
     "dev": "yarn build:backend && concurrently -k -n backend,frontend -c blue,green \"yarn start:backend\" \"nuxt-ts dev\"",
-    "start:backend": "NODE_ENV=development electron ../build/replay-app",
+    "start:backend": "cross-env CLEAR_LOGS=true NODE_ENV=development electron ../build/replay-app",
     "start:frontend": "nuxt-ts start",
-    "logs": "touch ~/Library/Logs/@secret-agent/replay-app/main.log && tail -f -100 ~/Library/Logs/@secret-agent/replay-app/main.log",
-    "logs:reset": "touch ~/Library/Logs/@secret-agent/replay-app/main.log && rm ~/Library/Logs/@secret-agent/replay-app/main.log"
+    "start": "electron ../build/replay-app",
+    "logs": "touch ~/Library/Logs/@secret-agent/replay-app/main.log && tail -f -100 ~/Library/Logs/@secret-agent/replay-app/main.log"
```

Comment:
I'm just now realizing that this ~/Library path shouldn't be here. Not sure we need to fix in this PR though.

Reply:
Yeah, I didn't find a cross environment way to fix this. I ended up leaving it as a utility, but took out of the build script